packages/manager/apps/telecom/src/app/telecom/telephony/repayments/repayments.html

```diff
@@ -0,0 +1,64 @@
+<section class="telephony-group-line-repayments">
+    <section class="widget-presentation">
+        <div class="row">
+            <div class="col-md-6">
+                <h4 data-translate="telephony_repayments_repayments"></h4>
+                <dl>
+                    <dt
+                        data-translate="telephony_repayments_pending_title"
+                    ></dt>
+                    <dd data-ng-bind="$ctrl.pendingAmountText"></dd>
```

Comment:
seems value is immutable, consider to have one time binding.
```suggestion
                    <dd data-ng-bind=":: $ctrl.pendingAmountText"></dd>
```